cmd/podman/varlink/io.podman.varlink

```diff
@@ -1112,6 +1112,10 @@ method ContainerRunlabel(runlabel: Runlabel) -> ()
 # ~~~
 method ListContainerMounts() -> (mounts: [string]string)
 
+method CopyTo(src: string, srcPath: string, srcfMode: int, destPath: string, extract: bool, pause: bool, ctrNameorId: string) -> (path: string)
```

Comment:
Methods should have comments, those will be copied into API.md when that file is generated.

Reply:
fixed